go.mod

```diff
@@ -107,6 +107,7 @@ require (
 	github.com/jonboulle/clockwork v0.2.2 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/kcp-dev/kcp/pkg/client v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
went bit onto a rabbit hole here. How does one update this with the real version before we do first release of the new module? 
https://github.com/kcp-dev/kcp/pull/2676#discussion_r1087148680

I suspect we need to do this first, release and do one more PR?

Reply:
This can stay as 0.0.0